src/components/Categories/Categories.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+  Box, Grid, Stack, Typography,
+} from "@mui/material";
+import { v4 as uuidv4 } from "uuid";
+import { ProductCard } from "./ProductCard";
+
+export const Categories = () => {
+  const categoryNames = ["Low Prices >", "All >", "Sports >"];
```

Comment:
These categories will be in the backend. So, they will be in the DB. This means, they will have the ids. 
```
const categoryNames = [
   {
      id: 1,
      name: 'Low prices'
   }
]
```

And this thing ">", should be in the JSX, not in the database

Reply:
done